tests/test_brighterFatterKernel.py

```diff
@@ -135,20 +135,20 @@ def test_quadratic(self):
         ptc.noise['amp 1'] = 5.0
 
         results = task.run(ptc, ['this is a dummy exposure'], self.camera, {'detector': 1})
-        expectation = np.array([[-3.12500e-06, -6.25000e-06, 3.12500e-06, 1.25000e-05,
-                                 3.12500e-06, -6.25000e-06, -3.12500e-06],
-                                [-6.25000e-06, -2.50000e-05, 6.25000e-06, 4.37500e-05,
-                                 6.25000e-06, -2.50000e-05, -6.25000e-06],
-                                [3.12500e-06, 6.25000e-06, 1.03125e-04, 2.50000e-04,
-                                 1.03125e-04, 6.25000e-06, 3.12500e-06],
-                                [1.25000e-05, 4.37500e-05, 2.50000e-04, 8.50000e-04,
-                                 2.50000e-04, 4.37500e-05, 1.25000e-05],
-                                [3.12500e-06, 6.25000e-06, 1.03125e-04, 2.50000e-04,
-                                 1.03125e-04, 6.25000e-06, 3.12500e-06],
-                                [-6.25000e-06, -2.50000e-05, 6.25000e-06, 4.37500e-05,
-                                 6.25000e-06, -2.50000e-05, -6.25000e-06],
-                                [-3.12500e-06, -6.25000e-06, 3.12500e-06, 1.25000e-05,
-                                 3.12500e-06, -6.25000e-06, -3.12500e-06]])
+        expectation = np.array([[1.55337977e-30, 3.13979520e-30, 4.60562292e-30, 5.19037630e-30,
+                                 4.60562292e-30, 3.13979520e-30, 1.55337977e-30],
+                                [3.07372386e-30, 6.40017812e-30, 1.00923202e-29, 1.15502594e-29,
+                                 1.00923202e-29, 6.40017812e-30, 3.07372386e-30],
+                                [4.34133757e-30, 9.45605038e-30, 1.43881780e-29, 1.39759388e-29,
+                                 1.43881780e-29, 9.45605038e-30, 4.34133757e-30],
+                                [4.83557602e-30, 1.06596290e-29, 1.71783205e-29, 1.55771296e-29,
+                                 1.71783205e-29, 1.06596290e-29, 4.83557602e-30],
+                                [4.34133757e-30, 9.45605038e-30, 1.43881780e-29, 1.39759388e-29,
+                                 1.43881780e-29, 9.45605038e-30, 4.34133757e-30],
+                                [3.07372386e-30, 6.40017812e-30, 1.00923202e-29, 1.15502594e-29,
+                                 1.00923202e-29, 6.40017812e-30, 3.07372386e-30],
+                                [1.55337977e-30, 3.13979520e-30, 4.60562292e-30, 5.19037630e-30,
+                                 4.60562292e-30, 3.13979520e-30, 1.55337977e-30]])
```

Comment:
I'm reasonably sure the values are correct.  I don't think the previous solution was calling the quadratic code correctly, and I think the values are so odd because the input data is made up.
I don't think the tolerance would have worked with the old values, as if it had, I likely would have left it alone.  I believe the test failed, and that's why I updated the solution expectation.